XU-TechnicalPaper.md

```diff
@@ -242,12 +242,31 @@ placeOrder(LTC/ETC, 0, 10)
 -------------
 The Lightning Network and the Raiden Network take care of security on the payment channel layer. This includes resolving disputes over channel states on the underlying blockchain and disincentivizing malicious behavior by POS-style punishment (a dishonest party will lose all her funds to the honest party). It especially also includes security when routing payments through intermediary hops. We use atomic swaps to bind two payments of a trade together, so that one party cannot steal funds after receiving the first part of the trade. Both payments in a trade have to complete successfully to be valid and spendable. Exchange Union will be secured against malicious order information in a POS-style manner, to disincentivize market manipulation by sending out false order information. As mentioned above, this is not part of the PoC and will be fleshed out in a later stage.
 
-`xud`'s ID Model
+LockDuration
+=============================================================
+
+Let's take a look at the steps of a typical swap flow from a technical perspective, and how `lockDuration` plays an important role to guarantee security in a swap. `lockDuration` is the time (counted in blocks) after which a payment *expires* when it's not resolved by revealing the according preimage. As described in the swap protocol above, we have a taker, a maker and two payments. Payment "one" taker --> maker and payment "two" maker --> taker. The taker creates and controls the preimage. To make swaps secure, we have to ensure, that the `lockDuration` of payment one is significantly **longer** than the `lockDuration` of payment two, which means payment one expires a decent amount of time **later** than payment two. The maker can specify how much time has to lie between the expiries with the variable `lockBuffer`. This buffer is needed, to give the maker enough time to redeem payment one, after the taker redeemed payment two by revealing the preimage. The introduced `lockBuffer` prevents scenarios where the maker can't redeem payment one after experiencing a software malfunction or loss of connection after the taker successfully redeemed payment two. In our protocol, the default value for `lockBuffer` is an equivalent of one day.
+
+Let's take a look at what happens in detail after a match for a LTC/BTC swap was found:
+1. Taker (outbound BTC, inbound LTC) runs [`QueryRoutes`](https://github.com/ExchangeUnion/xud/pull/650) as a basic check to see if a BTC route to the maker exists.
+2. Maker (outbound LTC, inbound BTC) receives a `SwapRequestPacket` from the taker. 
+3. Maker sends [`acceptDeal`](https://github.com/ExchangeUnion/xud/blob/master/lib/swaps/Swaps.ts#L413) if order is still available.
```

Comment:
Maybe we should exclude links to code because it is prone to change.

Reply:
Just did that in https://github.com/ExchangeUnion/Docs/commit/4dabb721625e1feeb84735fa0cbc6520320336ab. Not sure why it doesn't update here in the diff.